YamlSplitter/Models/MarkdownProperty.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Tools.YamlSplitter.Models
+{
+    using System.Collections.Generic;
+    using System.Text;
+
+    public class MarkdownProperty
+    {
+        public string OPath { get; set; }
+        public string Content { get; set; }
+        public Dictionary<string, object> Metadata { get; set; }
+
+        public override string ToString()
+        {
+            StringBuilder sb = new StringBuilder();
```

Comment:
Metadata is ignored?

Reply:
yes, for now, they don't plan to support property level yaml header